README.md

```diff
@@ -62,6 +62,17 @@ mysql -u root -p  Bassa < Bassa.sql
 
 3. Open components/core/DBCon.py and setup database username and password.
 
+The required changes in DBCon.py would look like :
+
+If the environment variables are being used, modify as following :
+```
+_db=MySQLdb.connect("db", os.environ.get('YOUR_DB_USERNAME'), os.environ.get('YOUR_DB_PASSWORD'), os.environ.get('Bassa'))
```

Comment:
Shall we change `os.environ.get('Bassa')` into `os.environ.get('YOUR_DB_NAME')`?